pages/index.html

```diff
@@ -139,7 +139,7 @@ <h3 class="usa-graphic-list-heading">Reuse reuse reuse.</h3>
 <section class="usa-section usa-section-alt usa-section-examples">
   <div class="usa-grid">
     <div class="usa-intro">
-      <h2>For example...</h2>
+      <h2>For example …</h2>
```

Comment:
@carodew @mollieru could we say "Examples" or something else that avoids the ellipses all together? I think it would look cleaner that way. /cc @maya